src/ansys/fluent/core/launcher/pyfluent_enums.py

```diff
@@ -164,7 +164,9 @@ def _get_running_session_mode(
         try:
             session_mode = FluentMode.get_mode(
                 "solver"
-                if fluent_connection.scheme_eval.scheme_eval("(cx-solver-mode?)")
+                if fluent_connection._connection_interface.scheme_eval.scheme_eval(
+                    "(cx-solver-mode?)"
```

Comment:
Can we replace this with a public method of `_ConnectionInterface` class?

Reply:
yes, done that. Thank you.